config.json

```diff
@@ -14,14 +14,7 @@
     "indent_style": "space",
     "indent_size": 4
   },
-  "files": {
```

Comment:
The configlet creates is messing with this file. Should we make one commit that has it according to configlet, so it is not scrambled every time we use it?

Reply:
Do we want to retain the empty config? If people are using the configlet to add exercises, is there value to it?

Reply:
I addded files in the leap exercises. It's nice.